frontend/src/component/ProtectedRoute.tsx

```diff
@@ -0,0 +1,12 @@
+import {isLoggedIn} from "../service/auth";
+import {Props} from "../constants";
+
+export const ProtectedRoute = (props: Props) => {
+    if (!isLoggedIn()) {
+        return (
+            <div>Not logged in</div>
+        );
+    }
+
+    return <div children={props.children} />
```

Comment:
```suggestion
    return <div>{props.children}</div>
```
<kbd>div</kbd> isn't a void tag.